client/src/Component/navbar/Navbar.js

```diff
@@ -3,11 +3,12 @@ import "./NavBar.css";
 import {Link} from 'react-router-dom';
 import AddAQuestion from "../AddAQuestion/AddAQuestion";
 import { SearchBar } from '../searchbar/SearchBar';
-import Logo from '../kurakoo-logo.png';
+import Logo from '../components/images/kurakoo-logo.png';
```

Comment:
Remove Un-necessary changes.

Reply:
In the second commit named scroll button added i have only two changes.